### ISSUE TITLE
Improve styling further: 

### ISSUE THREAD

**Issue by asadsid95** [asadsid95/port-folio#12]
- [x] Everything that is interact-able should be a button or a link (no spans, it's bad for accessibility)
   - Convert spans tags to buttons or links
 
- [x] Make project images in landing page bigger
    - make better use of white space

- [x] Add a favicon  

- [x] Add formatting in blogs 
- [x] Add description of creating the project for each project page
